content/general/concepts/cyberattack/cyberattack.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'Cyberattack'
+Description: 'Cyberattacks are attempts to disrupt or disable computer systems or steal valuable data.'
+Subjects:
+  - 'Computer Science'
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Encoding'
+  - 'Validation'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/securing-express-applications'
+---
+
+<link rel="canonical" href="https://www.codecademy.com/resources/blog/what-is-a-cyber-attack/" />
+
+Cyberattacks are attempts to disrupt or disable computer systems or steal valuable data. They can compromise sensitive data, including:
+
+- Personal information like Social Security numbers, birthdates, or medical information.
```

Comment:
```suggestion
- Personal information like social security numbers, birthdates, or medical information.
```

Reply:
I'm actually thinking of ditching the bit about Social Security b/c it might not make sense for readers outside the US who are reading these entries. Maybe I should switch with "government-issued documents"?